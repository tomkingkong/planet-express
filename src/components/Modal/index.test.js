import React from 'react';
import { shallow } from 'enzyme';

import { Modal } from '.';

describe('Modal', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Modal />);
    expect(wrapper).toMatchSnapshot();
  });
});