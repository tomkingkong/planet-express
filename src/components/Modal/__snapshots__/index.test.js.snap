// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal should match snapshot 1`] = `
<div
  className="Confirm"
>
  <h4
    className="confirm__header"
  >
    Confirm destination:
  </h4>
  <div
    className="confirm__options"
  >
    <button
      className="confirm__yes"
    >
      Confirm
    </button>
    <button
      className="confirm__no"
    >
      Decline
    </button>
  </div>
</div>
`;
